packages/react/focus-scope/src/FocusScope.test.tsx

```diff
@@ -37,7 +37,7 @@ describe('FocusScope', () => {
     it('should focus the next element in the scope on tab', () => {
       tabbableFirst.focus();
```

Comment:
You need to wrap this in `act`. That might remove the need for the later `waitFor`.